mkat_fpga_tests/test_cbf.py

```diff
@@ -64,6 +64,26 @@ def test_channelisation(self):
         """TP.C.1.19 CBF Channelisation Wideband Coarse L-band"""
         test_chan = 1500
         expected_fc = self.corr_freqs.chan_freqs[test_chan]
+        fhosts = {}
+        xhosts = {}
+        dicts = {}
+        dicts['fhosts'] = {}
+        dicts['xhosts'] = {}
+        fengs = self.correlator.fhosts
+        xengs = self.correlator.xhosts
+
+        def check_fftoverflow_qdrstatus():
```

Comment:
This should be called get_fft..., not check_fft..., since we are not actually checking anything here.